engine/flink/avro-util/src/test/scala/pl/touk/nussknacker/engine/avro/helpers/KafkaAvroSpecMixin.scala

```diff
@@ -34,8 +34,7 @@ trait KafkaAvroSpecMixin extends FunSuite with KafkaWithSchemaRegistryOperations
 
   protected def confluentClientFactory: ConfluentSchemaRegistryClientFactory
 
-  protected lazy val schemaRegistryProvider: ConfluentSchemaRegistryProvider =
-    ConfluentSchemaRegistryProvider(confluentClientFactory)
+  protected val schemaRegistryProvider: ConfluentSchemaRegistryProvider
```

Comment:
Can we have ConfluentSchemaRegistryProvider.avroPayload as a default provider that can be overriden? Alao why no to lazy val? lazy val is helpful in case when you have non-trival inheritance and you want to avoid some initialization order problems...

Reply:
Reverted to lazy val and used explicit `avroPayload`.